grpc-circuitbreaker-utils/src/main/java/org/hypertrace/circuitbreaker/grpcutils/CircuitBreakerConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package org.hypertrace.circuitbreaker.grpcutils;
+
+import java.time.Duration;
+import lombok.Builder;
+import lombok.Setter;
+import lombok.Value;
+
+@Value
+@Builder
+@Setter
```

Comment:
nit - shouldn't be a setter _and_ value - value is immutable, builder allows setting.